components/expenses/ExpensePayeeDetails.js

```diff
@@ -163,13 +164,13 @@ const ExpensePayeeDetails = ({ expense, host, isLoading, borderless, isLoadingLo
               <Avatar collective={host} radius={24} />
               <Span ml={2} color="black.900" fontSize="12px" fontWeight="bold" truncateOverflow>
                 {collective && (collective.isApproved || collective.id === host.id) ? (
-                  host.name
+                  concatenateDisplayNameWithLegalName(host.legalName, host.name)
```

Comment:
:ice_cream:  Somehow I would find it simpler to have a signature like:

```suggestion
                  formatAccountName(host, { showLegal: true })
```

But that's a personal pref, no requested change.

Reply:
I've renamed the function to `formatAccountName`; but I didn't quite understand the `{ showLegal: true }`. Why would we pass in an object ? And also isn't it that the whole intention of the function is to show the legal name so not sure why we need to pass in a `showLegal` arguement? 🤔

Reply:
> Why would we pass in an object

To make it explicit what this argument is, and allow for more customization options if needed in the future. But that's just a suggestion, other alternatives like `formatAccountName(account, showLegalName = true)` or `formatAccountNameWithLegalName` are valid too.